### PR TITLE
added disable_cookies as default for api actions

### PR DESCRIPTION
**Pull request by confact**
Added disable cookies as default for API actions

### CODE REVIEW

File: src/web_app_skeleton/src/actions/api_action.cr.ecr [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Include modules and add methods that are for all API requests
 abstract class ApiAction < Lucky::Action
+  disable_cookies
```
Comment (jwoertink) [inline src/web_app_skeleton/src/actions/api_action.cr.ecr:3]: Can we add a little comment on what this is removing? It removes sessions too, doesn't it?

**Comment by confact** [comment]
@jwoertink Now API action's current user is using JWT as default and should not need cookies and sessions. If they want to use API action with cookies and sessions they would need to use the module as in the Browser action anyway.

Like in API Action we include:
```
include Api::Auth::Helpers
include Api::Auth::RequireAuthToken
```
But in browser action we include:
```
include Auth::RequireSignIn
```

I can add comments for it though to know why it is there and can be removed if they want to use session and cookies (they would need to change `Auth::Helpers` and `RequireAuthToken` as well)

File: src/web_app_skeleton/src/actions/api_action.cr.ecr (hunk 1 above)
Comment (confact) [reply src/web_app_skeleton/src/actions/api_action.cr.ecr:4]: Added comment @jwoertink - feel free to change it to something better :)

**Comment by jwoertink** [comment]
Ah, that makes sense. 👍

File: src/web_app_skeleton/src/actions/api_action.cr.ecr (hunk 1 above)
Comment (paulcsmith) [reply src/web_app_skeleton/src/actions/api_action.cr.ecr:4]: Thanks for catching this and adding it. It makes sense to disable cookies by default IMO! Is this right that "Cookie header will still be sent"? I thought the header wouldn't be sent if cookies are disabled?

Comment (confact) [reply src/web_app_skeleton/src/actions/api_action.cr.ecr:5]: @paulcsmith you are right. I must have been confused of the request headers. I change the comment now.

Feel free to merge it if it looks good :)

**Pull request merged**